scripts/generate_test_tsv.py

```diff
@@ -0,0 +1,206 @@
+                ('RawDataProtocols', 'Primer'), 
+                ('RawDataProtocols', 'RawDataConditions'), 
+                ('RawDataProtocols', 'SequencingMethod'), 
+                ('RawDataProtocols', 'TargetGene'), 
+                ('Results', 'ResultID'), 
+                ('Results', 'ResultsLocation'), 
+                ('ResultsProtocol', 'ResultsDatePerformed'), 
+                ('ResultsProtocol', 'ResultsProcessor'), 
+                ('ResultsProtocol', 'ResultsProtocolID'), 
+                ('ResultsProtocols', 'ResultsMethod'), 
+                ('ResultsProtocols', 'ResultsTool'), 
+                ('ResultsProtocols', 'ResultsToolVersion'), 
+                ('Sample', 'SampleID'),
+                ('Sample', 'SampleWeight'),
+                ('Sample', 'SampleWeightUnit'),
+                ('SampleProtocol', 'SampleDatePerformed'),
+                ('SampleProtocol', 'SampleProcessor'),
+                ('SampleProtocol', 'SampleProtocolID'),
+                ('SampleProtocol', 'SampleProtocolNotes'),
+                ('SampleProtocols', 'SampleConditions'),
+                ('SampleProtocols', 'SampleTool'),
+                ('SampleProtocols', 'SampleToolVersion'),
+                ('Specimen', 'SpecimenCollectionDate'),
+                ('Specimen', 'SpecimenCollectionMethod'),
+                ('Specimen', 'SpecimenCollectionTime'),
+                ('Specimen', 'SpecimenID'),
+                ('Specimen', 'SpecimenNotes'),
+                ('Specimen', 'SpecimenWeight'),
+                ('Specimen', 'SpecimenWeightUnit'),
+                ('StorageLocation', 'StorageInstitution'),
+                ('StorageLocation', 'StorageFreezer'),
+                ('Study', 'RelevantLinks'),
+                ('Study', 'StudyName'),
+                ('Study', 'StudyType'),
+                ('Type', 'SpecimenType'),
+                ('Type', 'UberonCodeType')]
+
+    
+    # Blank data columns
+    headerCount = 3
+    diff = len(df)-headerCount
+
+    for header in headers:
+        blank_column = df.copy(deep=True)
+        blank_column.loc[headerCount:len(df), header] = np.array([''] * diff)
+        
+        write_test_metadata(blank_column, '{}/blank_column_tests/{}/'.format(file_path, file_type), 'blank_{}.tsv'.format(header[1]))
+
+    # NA data columns
+    for header in headers:
+        na_column = df.copy(deep=True)
+        na_column.loc[headerCount:len(df), header] = np.array(['NA'] * diff)
+
+        write_test_metadata(na_column, '{}/na_column_tests/{}/'.format(file_path, file_type), 'na_{}.tsv'.format(header[1]))
+
+    # Other data columns
+    for header in headers:
+        other_column = df.copy(deep=True)
+        other_column.loc[headerCount:len(df), header] = np.array([{True if randrange(2) == 0 else False : True if randrange(2) == 0 else False} for i in range(diff)])
+
+        write_test_metadata(other_column, '{}/other_column_tests/{}/'.format(file_path, file_type), 'other_{}.tsv'.format(header[1]))
```

Comment:
```suggestion
        write_test_metadata(other_column, file_path / 'other_column_tests' / file_type, f'other_{header[1]}.tsv'))
```
Leverage pathlib for managing filepaths whenever possible.